pkg/template/resource_test.go

```diff
@@ -469,7 +469,7 @@ func Test_ResolveTrigger_error(t *testing.T) {
 		{
 			name: "error triggerbinding and triggertemplate",
 			trigger: bldr.Trigger("invalid-tt-name", "v1alpha1",
```

Comment:
Can we add a test that uses an embedded binding as well?

Reply:
Done.